Source/ASNetworkImageNode.mm

```diff
@@ -669,8 +670,7 @@ - (void)_lazilyLoadImageIfNecessary
 
         if (_delegateFlags.delegateDidLoadImageWithInfo) {
           ASDN::MutexUnlocker u(__instanceLock__);
-          ASNetworkImageNodeDidLoadInfo info = {};
-          info.imageSource = ASNetworkImageSourceFileURL;
+          auto info = [[ASNetworkImageLoadInfo alloc] initWithURL:URL sourceType:ASNetworkImageSourceFileURL downloadIdentifier:nil userInfo:nil];
```

Comment:
Since we already checked above that `URL` and `_URL` are still equal, while we were locked, I standardized all these references to use the local variable instead of the ivar, since I had no choice but to use the local variable here (post-unlock).